apy/anki.py

```diff
@@ -320,8 +320,11 @@ def list_cards(self, query, verbose=False):
         """List cards that match a query"""
         for cid in self.find_cards(query):
             c = self.col.getCard(cid)
-            question = html_to_screen(c.q()).replace('\n', ' ')
-            answer = html_to_screen(c.a()).replace('\n', ' ')
+            # Use the note raw values instead of the templated values c.q() and c.a()
```

Comment:
Seems sensible. I prefer an empty line above comments, and I think the comment could be improved to explain _why_ we should use raw values.

Reply:
I rewrote the comment using Anki vocabulary.
A card has templates and fields. We want to display the field content and not the template extra markup.